### ISSUE TITLE
prop_miss_case() bug?

### ISSUE THREAD

**Issue by earowang** [njtierney/naniar#232]
1?

``` r
naniar::prop_miss_case(tibble::tibble(x = NA))
#> Registered S3 methods overwritten by 'ggplot2':
#>   method         from 
#>   [.quosures     rlang
#>   c.quosures     rlang
#>   print.quosures rlang
#> [1] 0
```

<sup>Created on 2019-06-06 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by njtierney** [comment]
Current behaviour 
``` r
naniar::prop_miss_case(tibble::tibble(x = NA))
#> [1] 1
```

<sup>Created on 2019-10-21 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by njtierney** [comment]
Closed in #239 

Thanks for your help, @earowang !
